docker/compose/mozdef_base/Dockerfile

```diff
@@ -22,10 +23,10 @@ RUN \
                     python \
                     python-devel \
                     python-pip && \
-  chown -R mozdef:mozdef /opt/mozdef/ && \
   pip install virtualenv && \
   mkdir /opt/mozdef/envs/mozdef && \
-  mkdir /opt/mozdef/envs/mozdef/cron
+  mkdir /opt/mozdef/envs/mozdef/cron && \
```

Comment:
We shouldn't need to do this, the cron container should be responsible for creating that directory I would think.

Reply:
I'm going to defer addressing this one.